src/components/MarkerDetails.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import '../App.css';
+
+
+
+class MarkerDetails extends Component {
+    
+    render(){
+   
+    return(
+        
+        <div className="tooltip">
+            <img src= "/images/burger.jpg" alt="food" height='30px' width='30px'/> 
```

Comment:
Why burger.jpg

Reply:
The MarkerDetails.js file is the start to refactoring the markers as a class to be imported an used in the App.js file. I think what I had really intended to do was push these changes to the feature/mapbox branch and continue working on them until they were ready for a pull request into the develop branch.